README.md

```diff
@@ -89,25 +89,58 @@ The OS is designed to work with an MMU, thus, the target must have 4 swappable v
 
 ## Requirements
 
-At the moment, the project has only been assembled on Linux (Ubuntu 20.04 and 22.04), it should be compatible with Mac OS and Windows as long as you have:
+At the moment, the project has only been assembled on Linux (Ubuntu 20.04 and 22.04, Debian 12(bookworm)), it should be compatible with Mac OS and Windows as long as you have:
 
 * bash
 * git (to clone this repo)
 * make
 * python3 with pip3. Used for the `menuconfig`.
 * z88dk v2.2 (or later). Only its assembler, `z80asm`, is strictly required. The latest version of `z80asm` must be used as earlier versions don't have support for `MACRO`.
 
+Some commands are different between linux and windows, so we can't give you an example to build zos on windows, if you have any progress on it, please contact us.
+
 On Ubuntu, the following commands can be used to install the dependencies. They must be run as a user, not root!
 ```
 sudo apt update
 sudo apt install git python3 python3-pip
 pip3 install --ignore-installed --user kconfiglib
 ```
 
+On Debian, run this:
+```
+sudo apt update
+sudo apt install git python3
+sudo apt install python3-kconfiglib
+```
+
 For installing Z88DK, please [check out their Github project](https://github.com/z88dk/z88dk).
 
-## Configuring Zeal 8-bit OS
+If you got error like this:
```

Comment:
This isn't necessary, it is an issue related to z88dk installation more than Zeal 8-bit OS configuration